.github/workflows/test_dep_update.sh

```diff
@@ -7,9 +7,17 @@ git branch depupdate
 git checkout depupdate
 git reset --hard origin/master
 json="$(gh pr list --search "author:app/github-actions" --json headRefName,number)"
+if [[ $json == "[]" ]]; then
+    echo "No Dependency update PRs to merge"
+    exit
+fi
```

Comment:
If there's no updates then the CI action will fail without this, which is more common if we reduce the schedule from 24 hours.